src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,271 @@
+            System.out.println(t.getTaskType() + t.getStatusIcon() + " " + t.getDescription());
+            if (tasks.size() > 1) {
+                System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+            } else {
+                System.out.println("Now you have " +tasks.size() + " task in the list.");
+            }
+        } catch (DukeException | IndexOutOfBoundsException e) {
+            printInvalidMessage("todo");
+        }
+    }
+
+    public static void addEvent(String taskToAdd) {
+        try {
+            validateInput(taskToAdd, "event");
+            int splitIndex = taskToAdd.indexOf('/');
+            String description = taskToAdd.substring(0, splitIndex);
+            String dateTime = taskToAdd.substring(splitIndex + 3);
+            Event e = new Event(description, dateTime);
+            tasks.add(e);
+            System.out.println("Got it. I've added this task:");
+            System.out.println(e.getTaskType() + e.getStatusIcon() + " " + e.getDescription() + " at(: "
+                    + e.getDateTime() + ")");
+            System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        } catch (DukeException | IndexOutOfBoundsException e) {
+            printInvalidMessage("event");
+        }
+    }
+
+    public static void addDeadline(String taskToAdd){
+        try {
+            validateInput(taskToAdd, "deadline");
+            int splitIndex = taskToAdd.indexOf('/');
+            String description = taskToAdd.substring(0, splitIndex);
+            String dateTime = taskToAdd.substring(splitIndex + 3);
+            Deadline d = new Deadline(description, dateTime);
+            tasks.add(d);
+            System.out.println("Got it. I've added this task:");
+            System.out.println(d.getTaskType() + d.getStatusIcon() + " " + d.getDescription() + " (by: "
+                    + d.getDateTime() + ")");
+            System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+
+        } catch (DukeException | IndexOutOfBoundsException e) {
+            printInvalidMessage("deadline");
+        }
+    }
+
+    public static void deleteTask(String inputCommand) {
+        int taskIndex = Integer.parseInt(inputCommand) - 1;
+        if (taskIndex < 0 || taskIndex >= tasks.size()) {
+            System.out.println("Task does not exists!");
+        } else {
+            System.out.println("Noted. I've removed this task:");
+            Task currentTask = tasks.get(taskIndex);
+            System.out.println(currentTask.getTaskType() + currentTask.getStatusIcon() + " "
+                    + currentTask.getDescription());
+            tasks.remove(taskIndex);
+            if (tasks.size() > 1) {
+                System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+            } else  {
+                System.out.println("Now you have " + tasks.size() + " task in the list.");
+            }
```

Comment:
Consider reducing the redundancy in the code by using the ternary operator (`?:`) to choose and print the singular/plural form of task.